src/tests/sqlsmith/src/expr.rs

```diff
@@ -169,16 +240,18 @@ fn make_overlay(exprs: Vec<Expr>) -> Expr {
     }
 }
 
-fn make_func(func_name: &str, exprs: &[Expr]) -> Function {
+// DINSTINCT , ORDER BY or FILTER is allowed in aggregation functions,
+fn make_func(func_name: &str, exprs: &[Expr], distinct: bool) -> Function {
```

Comment:
```suggestion
fn make_simple_func(func_name: &str, exprs: &[Expr]) -> Function {
```

Please separate them into different functions `fn make_agg_func`, `fn make_distinct_agg_func`. That'd be more readable than a bool flag.

Reply:
Ok, thank you very much. I forget that distinct are only allowed for aggregate functions. Then It makes more sense to separate them into agg and not agg. 

I will take a look into other parameters and implement (like over, order by ) in another issue (the issue I mentioned before and possibly PR).